libsplinter/src/rest_api/events.rs

```diff
@@ -61,17 +61,16 @@ impl<T: Serialize + Debug + Clone + 'static> EventDealer<T> {
     }
 
     /// Send message to all created WebSockets
-    pub fn dispatch(&mut self, msg: T) -> Result<(), EventDealerError> {
+    pub fn dispatch(&mut self, msg: T) {
         self.senders.retain(|sender| {
             if let Err(err) = sender.unbounded_send(MessageWrapper::Message(msg.clone())) {
-                warn!("Dropping sender due to error: {}", err);
+                error!("Dropping sender due to error: {}", err);
```

Comment:
I'm not sure if these should be logged as an error in the long run: they may create a bunch of error noise at shutdown.

Reply:
Reverted to warn